accessibility-checker-engine/help-v4/en-US/text_quoted_correctly.html

```diff
@@ -84,8 +87,8 @@ <h3 id="ruleMessage"></h3>
 
 ### Who does this affect?
 
- * People using a screen reader, including blind, low vision and neurodivergent people
- * People using text only, monochrome or braille displays
+ * People using a screen reader, including blind, low vision, and neurodivergent people
+ * People using text-only, monochrome, or Braille displays
```

Comment:
Lowercase was correct
```suggestion
 * People using text-only, monochrome, or braille displays
```